examples/app-example-androidjunitrunner/app/build.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'com.android.application'
+apply plugin: 'com.facebook.testing.screenshot'
+
+project.screenshots.customTestRunner = true
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.0"
+    defaultConfig {
+
+        multiDexEnabled true
+        applicationId "com.example.nadera.myapplication"
```

Comment:
@tdrhq Thanks for spotting that i made it more generic for now removed my name from it at least :) cheers